packages/adp-tooling/src/writer/options.ts

```diff
@@ -120,3 +164,36 @@ function getOpenSourceMiddlewares(config: AdpWriterConfig): CustomMiddleware<obj
         }
     ];
 }
+
+/**
+ * Get a list of required custom tasks for S4.
+ *
+ * @param config full project configuration
+ * @returns list of required tasks.
+ */
+function getAdpCloudCustomTasks(config: AdpWriterConfig & { target: AbapTarget }): CustomTask[] {
+    const user = 'env:ABAP_USERNAME';
```

Comment:
As mentioned in my other comment, this shouldn't be here. In the very worst case that there is a valid scenario for this, use at least `FIORI_TOOLS_USER` and `FIORI_TOOLS_PASSWORD` because they are available if any previous Fiori tools task or middleware relied on user/password.

Reply:
As discussed on the phone, please remove it.

Reply:
done